src/action.js

```diff
@@ -269,16 +280,14 @@ async function addComment(pullRequestNumber, body, reportName) {
   }
 }
 
-async function addCheck(body, reportName, sha, conclusion) {
-  const checkName = reportName ? reportName : "coverage";
-
+async function addCheck(checkName, body, checkTitle, sha, conclusion) {
   await client.rest.checks.create({
     name: checkName,
     head_sha: sha,
     status: "completed",
     conclusion: conclusion,
     output: {
-      title: checkName,
+      title: checkTitle,
```

Comment:
this distinction between check name and title is so that the title can be more descriptive and look like this
<img width="551" alt="Screen Shot 2021-12-22 at 6 23 29 PM" src="https://user-images.githubusercontent.com/39174181/147173477-2c49bc15-e443-4814-b2f8-24eadc848f6c.png">
: